tests/tests/eth_provider.rs

```diff
@@ -1421,3 +1421,44 @@ async fn test_transaction_by_hash(#[future] katana_empty: Katana, _setup: ()) {
         transaction.tx
     );
 }
+
+#[rstest]
+#[awt]
+#[tokio::test(flavor = "multi_thread")]
+async fn test_with_other_fields(#[future] katana: Katana, _setup: ()) {
+    let eth_provider = katana.eth_provider();
+    let transaction = katana.most_recent_transaction().unwrap();
+
+    // Retrieve receipts by block number
+    let receipts = eth_provider
+        .block_receipts(Some(alloy_rpc_types::BlockId::Number(BlockNumberOrTag::Number(
+            transaction.block_number.unwrap(),
+        ))))
+        .await
+        .unwrap()
+        .unwrap();
+    assert_eq!(receipts.len(), 1);
+    let receipt = receipts.first().unwrap();
+    assert_eq!(receipt.transaction_index, transaction.transaction_index);
+    assert_eq!(receipt.block_hash, transaction.block_hash);
+    assert_eq!(receipt.block_number, transaction.block_number);
+
+    // Add a custom field to the receipt to simulate a run out of resources
+    let mut receipt_with_other_fields = receipt.clone();
+    receipt_with_other_fields
+        .other
+        .insert("run out of resources".to_string(), serde_json::Value::from("run out of resources"));
```

Comment:
Here you can check here and you will understand

https://github.com/kkrt-labs/kakarot-rpc/blob/51fa1eb4a7d0a194432f63892f93a1dd36c4d8bf/src/tracing/mod.rs#L343

you inserted not the correct stuff I think. Because this idea here is to mimic exactly what we have as the output of the indexer inside the database so you should do something like:

```rust
let mut receipt_with_other_fields = receipt.clone();
    receipt_with_other_fields
        .other
        .insert("isRunOutOfRessources".to_string(), serde_json::Value::Bool(true));
```

Reply:
fixed, thanks